src/ocean/net/server/unix/UnixListener.d

```diff
@@ -19,6 +19,7 @@ module ocean.net.server.unix.UnixListener;
 import ocean.net.server.unix.UnixConnectionHandler;
 import ocean.net.server.SelectListener;
 import ocean.io.select.EpollSelectDispatcher;
+import ocean.text.util.StringC;
```

Comment:
I guess this import in no longer needed (but you might need to import `ocean.stdc.string` to get `strlen()`)

Reply:
Probably right about not needing the import any more. `strlen` was already in use in this file (which is why I switched techniques) so someone else must have already imported it for me :-)